### ISSUE TITLE
Quests - Classic - Scarlet Diversions - Western Plaguelands - Flag is totally unplantable; thus no quests in zone can be done.

### ISSUE THREAD

**Issue by Zelwindin** [dalaranwow/dalaran-wow#3996]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

In the Scarlet Diversions quest where one burns the tent and plants the flag (the quest that unlocks all other quests in Western Plaguelands for Horde) the flag cannot be planted.

Literally. I just walked over every spec of dirt in the Crusader Camp and in no place could the flag be planted.

Not being able to complete this quest makes the ENTIRE zone of Western Plaguelands unavailable to Horde players :( :( :(.

This quest worked fine a couple months ago, no idea what happened.

**Expected behaviour**: Tell us what should happen instead

The flag should be plantable, and therefore the quest be completable.

**Steps to reproduce the problem**:

1. Pick up the "Scarlet Diversions" quest from the Bulwark in eastern Tirisfal Glades.
2. Head to the Scarlet camp where one completes the quest.
3. Attempt to plant the flag. Observe that the quest is not completable.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Easily tested but can provide proof if necessary.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Scarlet Diversions - QuestID: 5096

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate https://github.com/dalaranwow/dalaran-wow/issues/3883

fix will be on live soon.

**Comment by Kinesoa** [comment]
Just an update- this quest is still unplayable. I'm hoping that this will be updated soon.

**Comment by Rushor** [comment]
The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Virmire** [comment]
Tried this on 28th June 2017, encountered the problem mentioned in the first post: flag can not be planted, despite burning the tent down.
